IHP/Job/Runner.hs

```diff
@@ -106,6 +108,9 @@ jobWorkerFetchAndRunLoop JobWorkerArgs { .. } = do
     let ?context = frameworkConfig
     let ?modelContext = modelContext
 
+    runningJobs <- Queue.fetchRunningJobs @job
+    mapM_ (Queue.jobDidFail' job "Job was running at startup") runningJobs
```

Comment:
What happens if two job workers are started?

Reply:
Oops, yeah, that shouldn't go there. I had something else there first, which worked with timeouts, where it actually made sense - but now it definitely shouldn't go here, I think. Will take a closer look